ee/clickhouse/views/test/__snapshots__/test_clickhouse_experiments.ambr

```diff
@@ -157,36 +150,29 @@
                 (SELECT *,
                         if(notEmpty(arrayFilter(x -> notEmpty(x), prop_vals)), prop_vals, ['']) as prop
                  FROM
-                   (SELECT aggregation_target,
-                           timestamp,
+                   (SELECT e.timestamp as timestamp,
+                           pdi.person_id as aggregation_target,
+                           pdi.person_id as person_id ,
```

Comment:
we certainly don't need this :D

Kinda don't wanna touch this PR anymore though so will fix in a follow up if OK